frontend/public/components/events.jsx

```diff
@@ -190,7 +190,7 @@ export class EventsList extends React.Component {
               {...this.props}
               key={kind}
               type={type}
-              kind={kind}
+              kind={kindForReference(kind)}
```

Comment:
This might have false positives if the same kind is present in different API groups. We probably need to change `EventStream` to work with the resource/version/kind string.